src/constants.js

```diff
@@ -0,0 +1,14 @@
+export const HEADER_THEMES = {
+  white: 'white',
+  green: 'green'
+};
+
+export const LOGO_THEMES = {
+  white: 'white',
+  green: 'green'
+};
+
+export const LOGO_THEME_COLORS = {
+  white: '#fff',
+  green: '#009623' // $color-queen-elizabeth
```

Comment:
Yeah, because of this I like @lauriejones `fill` method better - its in CSS and allows us to use the actual variable rather than duplicating it here.